### PR TITLE
SITL: add jsbsim_ext ("external JSBSim") model

### PR DESCRIPTION
**Pull request by spernsteiner**
This branch adds a new aircraft model `jsbsim_ext` for the SITL simulator.  This is similar to the existing `jsbsim` model, but connects to an existing `jsbsim_proxy` process instead of starting JSBSim itself.  This allows JSBSim to run on a different machine from the SITL binary, which we need for the OpenSUT configuration.

The first commit adds the `jsbsim_ext` model as an exact copy of `jsbsim`, just renamed.  The second commit implements the new behavior.

Part of GaloisInc/verse-opensut#97

### CODE REVIEW

**Review by podhrmic — CHANGES_REQUESTED** [review]
Please push these changes into a new branch in https://github.com/galoisinc/ardupilot/

I opened an IT ticket to get you access, hopefully it will happen soon.

**Comment by spernsteiner** [comment]
> Please push these changes into a new branch in https://github.com/galoisinc/ardupilot/

That's a public repo - did we finally get DISTAR approval for VERSE?  I haven't heard any update on that in a while

**Comment by podhrmic** [comment]
~~Good point. Let me see.~~ You have write access to https://github.com/galoisinc/ardupilot/ now.

DISTAR is not ready yet, thus we will need to use this repo and move it over later.

**Comment by spernsteiner** [comment]
> we will need to use this repo and move it over later.

Is this good to merge, then?

**Review by podhrmic — APPROVED** [review]

**Comment by podhrmic** [comment]
Yes, just approved.

**Pull request merged**
